chroot-etc/sommelierrc

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+# Copyright (c) 2021 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Based on https://chromium.googlesource.com/chromiumos/containers/sommelier/+/refs/heads/master/sommelierrc
+
+if hash xsetroot > /dev/null 2>&1; then
```

Comment:
just `hash xsetroot 2>/dev/null`

Reply:
commit 41dfd2e7fed7f2e006c7c95d29a5cd0f09e18b90

`sommelierrc` is now pulled from upstream.